components/side-drawer/stories/side-drawer-dismissible-and-modal.stories.js

```diff
@@ -0,0 +1,92 @@
+import { html } from 'lit-element';
+import { argTypes } from './arg-types.js';
+import { spread } from '@open-wc/lit-helpers';
+import '@vonage/vwc-side-drawer/vwc-side-drawer.js';
+import '@vonage/vwc-list/vwc-list-expansion-panel.js';
+import '@vonage/vwc-text/vwc-text.js';
+
+export default {
+	title: 'Alpha/Components/Side Drawer',
+	component: 'vwc-side-drawer',
+	argTypes
+};
+
+const SideDrawerTemplate = args => html`
+<vwc-side-drawer id="side-drawer" ...=${spread(args)} @click="${onClick}">
+	<span slot="top-bar">
+		<vwc-icon type="vonage-mono"></vwc-icon>
+		<vwc-text font-face="body-1-bold"> VONAGE</vwc-text>
+	</span>
+	<span>
+		<vwc-list innerRole="navigation" innerAriaLabel="Primary navigation" itemRoles="link">
+			<vwc-list-item shape="rounded" graphic="icon">
+				<vwc-icon slot="graphic" type="home-line"></vwc-icon>1st level item
+			</vwc-list-item>
+			<p>
+				<vwc-text font-face="body-2-bold">SECTION TITLE</vwc-text>
+			</p>
+			<vwc-list-item shape="rounded" graphic="icon">
+				<vwc-icon slot="graphic" type="chat-line"></vwc-icon>1st level item
+			</vwc-list-item>
+			<vwc-list-expansion-panel open>
+				<vwc-list-item slot="header" shape="rounded" graphic="icon">
+					<vwc-icon slot="graphic" type="chat-line"></vwc-icon>1st level item
+				</vwc-list-item>
+				<vwc-list-expansion-panel open>
+					<vwc-list-item slot="header" shape="rounded">2nd level item</vwc-list-item>
+					<vwc-list-item shape="rounded">3rd level item</vwc-list-item>
+					<vwc-list-item shape="rounded">3rd level item</vwc-list-item>
+				</vwc-list-expansion-panel>
+			</vwc-list-expansion-panel>
+			<p>
+				<vwc-text font-face="body-2-bold">SECTION TITLE</vwc-text>
+			</p>
+			<vwc-list-expansion-panel>
+				<vwc-list-item slot="header" shape="rounded" graphic="icon">
+					<vwc-icon slot="graphic" type="chat-line"></vwc-icon>1st level item
+				</vwc-list-item>
+				<vwc-list-item shape="rounded">2nd level item</vwc-list-item>
+				<vwc-list-item shape="rounded">2nd level item</vwc-list-item>
+			</vwc-list-expansion-panel>
+		</vwc-list>
+	</span>
+	<span slot="app-content">
```

Comment:
I'm not sure span is the right element. Because the content may contain various elements that can also be block elements I think Div is better